addons/im_livechat/static/src/models/thread_view_topbar/thread_view_topbar.js

```diff
@@ -0,0 +1,16 @@
+/** @odoo-module **/
+
+import { registerInstancePatchModel } from '@mail/model/model_core';
+
+registerInstancePatchModel('mail.thread_view_topbar', 'im_livechat/static/src/models/thread_view_topbar/thread_view_topbar.js', {
```

Comment:
Note that `model_core` functions and model names have changes in `saas-15.2`, so will have to adapt during forward-port.

`registerInstancePatchModel` => `patchRecordMethods` _(signature changed, so don't need filepath as 2nd argument anymore)_
`mail.thread_view_topbar` => `ThreadViewTopbar`